src/js/ui/menu.js

```diff
@@ -299,9 +305,17 @@ Menu.prototype.item = function(sectionIndex, itemIndex, item) {
   return this;
 };
 
-Menu.prototype.selection = function(callback) {
-  this._selections.push(callback);
-  simply.impl.menuSelection();
+Menu.prototype.selection = function(callback_or_sectionIndex, itemIndex) {
```

Comment:
There are some functions that can get pretty complex names with optional positional parameters. `callback_or_sectionIndex` can be `sectionIndex` to keep with the camelCase naming, and if it turns out to be a function, you can rebind it to another variable for clarity. `var callback = sectionIndex`. Generally the function signature should be the overload with the longest parameter list, and internally the values are renamed or shuffled around.